CrisisTextLine/Elmer
# Email from mandrill

#### What's this PR do?
Consolidates all messages related to Force melt into autoForceMelt.js
Modularizes the process of dropping melted users and emailing them 

#### Where should the reviewer start?
Follow the flow through autoForceMelt and then dropVoluntaryMelt.js

#### How should this be manually tested?
Change/Disable the email to your own mergeDataForSending() &&
Disabled the dropping functionality &&
Add console.logs to relevant parts

run the melt scripts and verify the users that would be affected.

#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=12&view=detail&selectedIssue=AT-176&quickFilter=44
#### Questions:
Is there a better place to call messageData.meltData ?
